src/makeCalculator.js

```diff
@@ -35,7 +35,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(target, value) {
+      target.result += value;
+    },
+    subtract(target, value) {
+      target.result -= value;
+    },
+    multiply(target, value) {
+      target.result *= value;
+    },
+    divide(target, value) {
+      target.result /= value;
+    },
+    operate(callback, value) {
+      callback(this, value);
```

Comment:
не передавай весь this в callback, тобі потрібний тільки this.result
```suggestion
      this.result = callback(this.result, value);
```

У функція потрібно буде прописати return, щоб цей код працював